renovate.json

```diff
@@ -10,7 +10,26 @@
     },
     {
       "matchPackageNames": ["php"],
-      "rangeStrategy": "replace"
+      "rangeStrategy": "replace",
+      "groupName": "PHP"
+    },
+    {
+      "matchFiles": [".platform.app.yaml"],
+      "matchPackageNames": ["openswoole/ext-openswoole"],
+      "extends": [":automergeDisabled", ":automergePr", ":label(Awaiting Maintainer Response)"],
+      "prBodyNotes": [
+        ":warning: CI does not verify if extension can be built successfully or compatible. Build log review at platform.sh is required."
+      ]
+    },
+    {
+      "matchFiles": [".platform.app.yaml"],
+      "matchPackageNames": ["php/php-src"],
```

Comment:
Will you also need to set `rangeStrategy` for this as it will be bump instead of replace?

Reply:
It is not composer require dependencies, so no it won't expand. 

Should strategy be explicitly specified as currently it uses default replace strategy?

Reply:
No you're right, I mistakenly thought you'd changed the default "rangeStrategy".